distributions/Dist_psa251.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+from base_distribution import BaseDistribution
+
+# intentional misspelling to avoid syntax errors
+lamda = 4.
```

Comment:
this is dangerous b/c if any other code that uses this defines lamba they will clash. Instead, in `__init__(self):` add a line `self.lamba=4.`. And then when you want to refer to that variable, use `self.lamba`